feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -193,24 +197,23 @@ $sapFioriContent_LabelColor: #6a6d70;
       vertical-align: top;
     }
   }
-  @include forVersion(1, 6) {
-    .cx-avatar-cell {
-      width: 65px;
-      vertical-align: top;
-      .cx-avatar {
-        width: 44px;
-        height: 44px;
-        line-height: 44px;
-        border-radius: 50%;
-        color: #fff;
-        text-align: center;
-        background: #9259cd;
-        text-transform: uppercase;
-        margin-top: 2px;
-        margin-inline-end: 10px;
-        margin-bottom: 2px;
-        margin-inline-start: 0;
-      }
+
+  .cx-avatar-cell {
```

Comment:
does this style works for every version?

Reply:
Done, Feature level has been added.